lib/style.dart

```diff
@@ -54,8 +54,18 @@ abstract class AppColors {
 }
 
 // TODO: Convert members to top level finals/consts or functions.
+// ignore: avoid_classes_with_only_static_members
```

Comment:
this ignore is not needed with change in the yaml config

Reply:
Yep cool. That actually wasn't me, but will remove 👍

Reply:
So need to confirm how to disable before i can remove this, otherwise lint rule gets triggered still.

Reply:
Also I don't think we want to do these TODOs that were here already:

// TODO: Convert members to top level finals/consts or functions.

Thoughts @maks ?

Reply:
Now that analysis_options.yaml fixed. Removed this line.👍